NEWS.md

```diff
@@ -5,6 +5,7 @@
 ## Polars R Package 0.20.0
 
 - Updated rust-polars to 0.43.1 (#1230).
+- Maintains level order when converting Enums to factors (#1252, @andyquinterom)
```

Comment:
I think you are adding it in the wrong place.

Reply:
Thanks for pointing this out